float32[] radius
# radius of detected circle
float32[] x_center
# x value of circles center 
float32[] y_center
# y value of circles center 
float32 red
# red value for markers 
float32 green
# gree value for markers 
float32 blue
# blue value for markers 
float32 alpha
# transperency value for markers 
string frame_name
#reference frame for drawing maps